debian/pbuilder-hooks/B92test-pkg

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# example file to be used with --hookdir
+#
+# run tests. Current directory is top of source-code.
+#
+# 2005, 2007 Junichi Uekawa
```

Comment:
What is the license for original file?

Reply:
We cannot use it - need to rewrite.

Reply:
We can use the file directly from the system, without including the source code.